src/client/Client.cc

```diff
@@ -3743,8 +3743,8 @@ void Client::send_cap(Inode *in, MetaSession *session, Cap *cap,
 				   flush,
 				   cap->mseq,
                                    cap_epoch_barrier);
-  m->caller_uid = in->cap_dirtier_uid;
-  m->caller_gid = in->cap_dirtier_gid;
+  m->caller_uid = -1;
+  m->caller_gid = -1;
```

Comment:
nit: could you paste the commit message as a comment here as well; it'll help explain things quicker